Extension/src/common.ts

```diff
@@ -776,6 +777,17 @@ export function promptReloadWindow(message: string): void {
     });
 }
 
+export async function CreateTempFileWithPostfix(postfix: string): Promise<tmp.FileResult> {
```

Comment:
This function returns a promise, but is not `async` because `await` is not used in it.